src/gfx/gfx.cpp

```diff
@@ -98,6 +98,9 @@ namespace ari
 		core::Map<uint32_t, ShaderDescShaderHandle> MaterialShaders;
 
 		static sx_mat4 g_mView, g_mProj, g_mViewProj, g_mWorld, g_mWorldViewProj;
+		static Viewport* g_Viewport = nullptr;
```

Comment:
Use pointer is not safe here

Reply:
why not? what if I set it as a const pointer?